python-lib/action.yml

```diff
@@ -9,7 +9,7 @@ inputs:
   public:
     description: "if true, push to public python package index pypi.org, otherwise use our private fury.io"
     required: false
-    default: false
+    default: "false"
```

Comment:
is it needed? yaml does't support boolean?
If so, we need to fix everywhere

Reply:
I have a linter running that told me the expected type is `string`, so I changed it. But not sure if it's required